gems/pending/spec/appliance_console/logfile_configuration_spec.rb

```diff
@@ -1,51 +1,119 @@
 require "appliance_console/logfile_configuration"
+require "tempfile"
 
 describe ApplianceConsole::LogfileConfiguration do
+  let(:seeded_miq_logs_conf) do
```

Comment:
I copied the idea and naming from: [database_maintenance_periodic_spec.rb](https://github.com/ManageIQ/manageiq/blob/master/gems/pending/spec/appliance_console/database_maintenance_periodic_spec.rb#L88)

I will change it here to `let(:original_miq_logs_conf)`